src/Agent.php

```diff
@@ -348,6 +370,24 @@ public function handleAgent($metadata)
             $input['tag'] = $metadata['tag'];
         }
 
+        if (isset($_SERVER['REMOTE_ADDR'])) {
+            if ($ip_binary = inet_pton($_SERVER['REMOTE_ADDR'])) {
```

Comment:
As discussed, we cannot rely on that; it will be wrong in czse of uploading an XML file, or if there are proxies between agent and glpi.